modules/ocf_desktop/manifests/defaults.pp

```diff
@@ -4,10 +4,23 @@
     source  => 'puppet:///modules/ocf_desktop/xsession/mimeapps.list';
   }
 
+  file { '/opt/share/puppet/pdf-open':
```

Comment:
should go in `/usr/local/bin` probably, i don't think we should encourage use of `/opt/share/puppet` (as much as possible), it's sort of a dumping ground